### ISSUE TITLE
Field order in template is not consistent with stress period schema

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#50]
In GitLab by @Huite on Apr 12, 2019, 11:45

This, in `imod.run`:

```python
stress_period_schema = OrderedDict(
    [
        ("wel", {"systems": True}),
        ("drn", {"systems": True, "order": ("cond", "bot")}),
        ("riv", {"systems": True, "order": ("cond", "stage", "bot", "inff")}),
        ("ghb", {"systems": True, "order": ("cond", "head")}),
        ("rch", {"systems": False}),
        ("chd", {"systems": False}),
    ]
)
```

Is not consistent with the template!

```  
{%- set names = ["bhead", "cond", "ghbssmdens"] %}
```

Probably check all these orders just to be sure...

**Comment by Manangka** [comment]
In GitLab by @JoerivanEngelen on Apr 12, 2019, 12:52

created merge request !23 to address this issue

**Comment by Manangka** [comment]
In GitLab by @Huite on Apr 12, 2019, 14:55

Fixed by 4433a06ed2d174a469354c832c05882edb2c5aab

**Comment by Manangka** [comment]
In GitLab by @Huite on Apr 12, 2019, 14:55

closed
